testing/discovery_interface.py

```diff
@@ -67,7 +67,8 @@ def wait_for_discovery_launch():
     # Timeout of 25 seconds for launch
     if not wait_for_discovery_status():
         LOGGER.error("Couldn't launch Discovery, printing Docker logs:\n---\n")
-        docker_log_and_stop()
+        log_discovery()
+        shutdown_discovery()
```

Comment:
Any reasoning behind this change?

Reply:
It was in our local discovery tests code and it seemed like a good idea to include so I left it in.